SNLS_TrDLDenseG.cxx

```diff
@@ -1,177 +1,2 @@
-      if (strm) { *((std::ostream *) strm) << "delta now at max " << delta << std::endl ; }
-#endif
-   }
-#ifdef __cuda_host_only__
-   else 
-      if (strm) { *((std::ostream *) strm) << "incr delta to "    << delta << std::endl ; }
-#endif
-}
-
-__snls_hdev__
-bool TrDeltaControl::updateDelta
-(
-   void    * strm      ,
-   double  & delta     ,
-   double    res       ,
-   double    res_0     ,
-   double    pred_resid,
-   bool    & reject    , // reject_prev
-   bool      took_full , // use_nr
-   double    normfull  , // nr2norm
-   double  & rho       ) const
-{
-   bool  success       = true;
-   double actual_change = res - res_0;
-   double pred_change   = pred_resid - res_0;
-
-   if ( pred_change == 0e0 ) 
-   {
-      if ( delta >= _deltaMax ) {
-         // things are going badly enough that the solver should probably stop
-#ifdef __cuda_host_only__
-         if (strm) { *((std::ostream *) strm) << "predicted change is zero and delta at max" << std::endl; }
-#endif
-         success = false ;
-      }
-      else {
-#ifdef __cuda_host_only__
-         if (strm) { *((std::ostream *) strm) << "predicted change is zero, forcing delta larger" << std::endl; }
-#endif
-         delta = fmin( delta * _xiForcedIncDelta, _deltaMax );
-         // ierr = IERR_UPD_DELTA_PRZ_p; // do not report this as a failure
-      }
-   }
-   else 
-   {
-      rho = actual_change / pred_change;
-#ifdef __cuda_host_only__
-      if (strm) { *((std::ostream *) strm) << "rho = " << rho << std::endl; }
-#endif
-      if ( ( rho > _xiLG ) &&
-           ( actual_change < 0e0 ) &&
-           ( rho < _xiUG ) ) {
-         if ( ! took_full ) {
-            // increase delta
-            this->incrDelta(strm, delta);
-         }
-         // instead of truncating, just do not increase, as 
-         // truncating can cause trouble
-         // else { 
-         // 
-         //    double temp = delta;
-         //    double temp_b = normfull * _xiIncDelta;
-         //    if (temp_b < delta) {
-         //       delta = temp_b;
-#ifdef __cuda_host_only__
-         //       if ( _os != NULL ) {
-         //          _os << "took full step, delta truncated from " << temp << " to " << delta << std::endl;
-         //       }
-#endif
-         //    }
-         // }
-      }
-      else if ( ( rho < _xiLO ) ||
-                ( rho > _xiUO ) ) 
-      { 
-         // decrease delta
-         success = this->decrDelta(strm, delta, normfull, took_full);
-      }
-   } // pred_change == 0
-   
-   reject = false;
-
-   // do not make this >= , may have res and res_0 both zero and that is ok
-   if ( ( actual_change > 0e0 ) && 
-        ( _rejectResIncrease ) ) 
-   { 
-      // residual increasing, need to reject solution
-      // have already decreased delta (rho < 0)
-      reject = true;
-   }
-   
-   return success;
-}
-
-} // namespace snls
+//BLT spits out errors if we dont have a source file.
+//So we're just going to leave this as being empty
```

Comment:
Please put whitespace between the // and the text of the comment.

Please add a newline at the end of the file.

Reply:
Done